### ISSUE TITLE
Bump mode-streaming to 0.2.0

### ISSUE THREAD

**Issue by tritas** [faust-streaming/faust#209]
## Checklist

- [x] I have included information about relevant versions
- [x] I have verified that the issue persists when using the `master` branch of Faust.

## Steps to reproduce

Run the following in a new env:

```
pip install git+git://github.com/faust-streaming/mode@master
pip install git+git://github.com/faust-streaming/faust@master#egg=faust-streaming
```

## Solution

Bump ``mode-streaming`` dependency in ``requirements/base.txt``

**Comment by taybin** [comment]
We're waiting to regain access to the pypi account for mode-streaming.  While mode-streaming has been *tagged* 0.2.0, we haven't been able to put a new release on pypi yet.  This is blocking other PRs and is very frustrating for all of us.

**Comment by tritas** [comment]
I see, let's wait for the good people at PyPI then.

**Comment by dennis-weyland-by** [comment]
@taybin since this is quite old now: Would it make sense to create a new pypi account and release mode under a new name? Yes, this solution is far from ideal but probably better than to wait for months until you gain the credentials for mode-streaming :/

**Comment by AntonOfTheWoods** [comment]
Bump?

**Comment by dennis-weyland-by** [comment]
@AntonOfTheWoods https://pypi.org/project/mode-streaming/ seems like they got the pypi Account back

**Comment by AntonOfTheWoods** [comment]
@dennis-weyland-by , nice!. So this can be closed, we can bump, and then use python 3.10?
